backend/src/main/java/com/woowacourse/levellog/prequestion/application/PreQuestionService.java

```diff
@@ -0,0 +1,121 @@
+package com.woowacourse.levellog.prequestion.application;
+
+import com.woowacourse.levellog.common.exception.InvalidFieldException;
+import com.woowacourse.levellog.common.exception.UnauthorizedException;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.levellog.domain.LevellogRepository;
+import com.woowacourse.levellog.levellog.exception.LevellogNotFoundException;
+import com.woowacourse.levellog.member.domain.Member;
+import com.woowacourse.levellog.member.domain.MemberRepository;
+import com.woowacourse.levellog.member.exception.MemberNotFoundException;
+import com.woowacourse.levellog.prequestion.domain.PreQuestion;
+import com.woowacourse.levellog.prequestion.domain.PreQuestionRepository;
+import com.woowacourse.levellog.prequestion.dto.PreQuestionDto;
+import com.woowacourse.levellog.prequestion.exception.PreQuestionNotFoundException;
+import com.woowacourse.levellog.team.domain.Participant;
+import com.woowacourse.levellog.team.domain.ParticipantRepository;
+import com.woowacourse.levellog.team.domain.Team;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class PreQuestionService {
+
+    private final PreQuestionRepository preQuestionRepository;
+    private final LevellogRepository levellogRepository;
+    private final MemberRepository memberRepository;
+    private final ParticipantRepository participantRepository;
+
+    @Transactional
+    public Long save(final PreQuestionDto request, final Long levellogId, final Long memberId) {
+        final Levellog levellog = getLevellog(levellogId);
+        final Member questioner = getMember(memberId);
+
+        validateSameTeamMember(levellog.getTeam(), questioner);
+
+        return preQuestionRepository.save(request.toEntity(levellog, questioner))
+                .getId();
+    }
+
+    public PreQuestionDto findById(final Long preQuestionId, final Long levellogId, final Long memberId) {
+        final PreQuestion preQuestion = getPreQuestion(preQuestionId);
+        final Levellog levellog = getLevellog(levellogId);
+        final Member questioner = getMember(memberId);
+
+        validateLevellog(preQuestion, levellog);
+        validateMyQuestion(preQuestion, questioner);
```

Comment:
```suggestion
        preQuestion.validate(levellog, questioner);
```
객체에게 메시지를 던져볼까요? (메서드 이름은 적절하게 변경해주세요^^)

Reply:
수정해봤습니다!